src/collective/fingerpointing/browser/logview.pt

```diff
@@ -21,9 +21,73 @@
     </metal:title>
 
     <metal:content-core metal:fill-slot="content-core">
-      <pre tal:condition="view/available" tal:content="view/get_audit_log" />
-
-      <p tal:condition="not:view/available" i18n:translate="">
+      <tal:block tal:condition="python:view.available"
+                 tal:define="dummy python:view.build_audit_log()">
+        <form method="get"
+              action=""
+              tal:attributes="action python:view.base_url">
+          <div class="field">
+            <input type="hidden"
+                   name="size"
+                   value=""
+                   tal:attributes="value python:view.size" />
+            <input type="text"
+                   name="query"
+                   value=""
+                   class="searchButton"
+                   placeholder="Enter a search term."
+                   i18n:attributes="placeholder"
+                   tal:attributes="value python:view.query" />
+          </div>
+          <div class="formControls button-field context">
+            <input type="submit" value="filter" i18n:attributes="filter"/>
```

Comment:
what about a `Clear filter` button?

Reply:
like the classical form reset?

Reply:
yes, currently to see again all events I have to clear the input and press the button; that can be done in one step with a clear filter option.